src/gt4py/next/embedded/nd_array_field.py

```diff
@@ -312,6 +317,7 @@ class NdArrayConnectivityField(  # type: ignore[misc] # for __ne__, __eq__
     NdArrayField[common.DimsT, core_defs.IntegralScalar],
 ):
     _codomain: common.DimT
+    _skip_value: Optional[core_defs.IntegralScalar]
```

Comment:
Out of scope of this PR but just to not forget: it doesn't make sense to use a property for plain attribute access just because a property was defined in a parent class. We could add a simple utility to eve to disable parent properties by overwriting them with a non-data descriptor like this:
```
def disable_property(default=...) -> functools.cached_property:
    return functools.cached_property(lambda _: default)
```
then this should work:
```suggestion
    codomain: common.DimT = disable_property()
    skip_value: Optional[core_defs.IntegralScalar] =  disable_property()
```
Inside a dataclass, it would also work to define a field with a default like this:
```
    codomain: common.DimT = dataclasses.field(default=...)
```
although this would change the semantics by making `codomain` optional, where `disable_property` wouldn't affect the behavior of the dataclass at all.